web/src/app/lists/ControlledPaginatedList.tsx

```diff
@@ -67,6 +67,7 @@ export interface ControlledPaginatedListAction {
 
   // label to display (e.g. "Close alerts")
   label: string
+  ariaLabel?: string
```

Comment:
We already have label text, `ariaLabel` is redundant -- we should re-use the existing OR if there is already a `title` attribute in the markup, we don't need to override it with `aria-label` as a screen reader will already read the title.

I'd rather make the label more descriptive if possible than always defining two descriptions for the same action -- where possible. (e.g. The tooltip/label could just say "Close Alerts" instead of just "Close" which would work for visual and keyboard users)

Reply:
I think that `aria-label` can be justified here as the tooltip should be short and concise for the majority of users (e.g. the current Ack, Close, Escalate), as they can see the relation next to the checkbox element. Though for screen readers I think it's acceptable they we add additional context to help them out, e.g. Acknowledge Selected Alerts, since we aren't limited on screen space.

Reply:
Instead, can we indicate it's a region (e.g. of actions) which will give context to "Close" "Ack" etc..?

I'd like to move away from having two textual descriptions of things, esp. in an API, and think it would be worthwhile to find a way to make it clearer without having to rely on aria-label  overriding the text already on the page.